osu.Game/OsuGame.cs

```diff
@@ -90,11 +90,18 @@ public class OsuGame : OsuGameBase, IKeyBindingHandler<GlobalAction>
 
         private IdleTracker idleTracker;
 
+        private ConfineMouseTracker confineMouseTracker;
+
         /// <summary>
         /// Whether overlays should be able to be opened game-wide. Value is sourced from the current active screen.
         /// </summary>
         public readonly IBindable<OverlayActivation> OverlayActivationMode = new Bindable<OverlayActivation>();
 
+        /// <summary>
+        /// Whether gameplay is currently active.
+        /// </summary>
+        public readonly Bindable<bool> IsGameplay = new BindableBool();
```

Comment:
```suggestion
        /// <summary>
        /// Whether the local user is currently interacting with the game in a way that should not be interrupted.
        /// </summary>
        public readonly Bindable<bool> LocalUserPlaying = new BindableBool();
```

rename other usages / privates to match.